### PR TITLE
Update GenomIO to avoid too many positional arguments

### PR DESCRIPTION
**Pull request by JAlvarezJarreta**
But ignore the issue in unit tests, since it is not relevant in that case.

### CODE REVIEW

File: src/python/tests/fasta/test_chunk.py [hunk 1]
```diff
@@ -12,12 +12,8 @@
...
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Unit testing of `ensembl.io.genomio.fasta.chunk` module.
-
-Typical usage example::
-    $ pytest test_chunk.py
-
-"""
+"""Unit testing of `ensembl.io.genomio.fasta.chunk` module."""
+# pylint: disable=too-many-positional-arguments
```
Comment (ens-LCampbell) [inline src/python/tests/fasta/test_chunk.py:16]: Slightly confused on adding this comment line, does this disable TMPA ?

Comment (ens-LCampbell) [reply src/python/tests/fasta/test_chunk.py:16]: Obvs same applies to other insertions of this line, just to clarify thanks

**Review by ens-LCampbell — APPROVED** [review]
Looks solid thanks Jorge :) Approved, just wanting clarification on tests pylint comment lines added

File: src/python/tests/fasta/test_chunk.py (hunk 1 above)
Comment (JAlvarezJarreta) [reply src/python/tests/fasta/test_chunk.py:16]: Yes, this disables that particular check for the whole file. I would have preferred to disable it completely for the `tests` folder, but `pylint` does not have such feature, so it has to be manually introduced in each test file.

Comment (ens-LCampbell) [reply src/python/tests/fasta/test_chunk.py:16]: Oh ok ok just checking as I wasn't sure if that enabled or disabelled the check thanks for clearing that up. Merge approved from my end

**Pull request merged**